test/integration/cart_integration_test.rb

```diff
@@ -229,4 +284,23 @@ class CartIntegrationTest < ActionDispatch::IntegrationTest
     assert page.has_content?("$24.00")
     assert page.has_content?("coffee")
   end
+
+  test "an authenticated user can delete a pending loan from their cart" do
+    authenticated_user = create(:user)
+    ApplicationController.any_instance.stubs(:current_user).
+      returns(authenticated_user)
+    tenant = create(:tenant)
+    tenant.projects << create(:project)
+
+    visit "/#{tenant.organization}"
+    within(".row") do
+      click_link_or_button("Lend")
+    end
+    click_link_or_button("Delete")
+
+    within('#pending_loans') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.